### ISSUE TITLE
Para Module but still no BT

### ISSUE THREAD

**Issue by Livewire85** [pfeerick/lang-firmwares#6]
Hi, I have installed a para module on the Aux1 port inside the Radiomaster Zorro, After installing this firmware im still not getting options for Bluetooth, A friend also did his radio (not a Zorro) and said that his Aux1 was replaced with Bluetooth, My Aux1 options dont seem to have changed.

https://github.com/pfeerick/lang-firmwares/assets/17522187/fd845a3f-1013-4772-846a-62017344712f

**Comment by puresims** [comment]
Yes, no bluetooth enabled it seems in "zorro-bluetooth-lang-v2.9.0.zip"

![378250034_1687864305053054_3449403785951973846_n](https://github.com/pfeerick/lang-firmwares/assets/80000593/2a22a4e3-4fd6-4c75-b23f-7817e9352141)
